src/Dependencies.jl

```diff
@@ -92,8 +100,10 @@ struct Dependency <: AbstractDependency
     build_version::Union{VersionNumber,Nothing}
     compat::String  # semver string for use in Project.toml of the JLL
     platforms::Vector{<:AbstractPlatform}
+    top_level::Bool
```

Comment:
Document this argument?  You may also want to add it to the other `Dependency` methods.

Reply:
Let me know if the docs are to detailed.

Reply:
I think it's fine :slightly_smiling_face:  Thanks!